src/images/badges/ctrl-c-ctrl-v.svg

```diff
@@ -11,29 +11,29 @@
 <g>
```

Comment:
Type looks to not be center aligned, and the R for the first `CTRL` seems to be of the wrong typeface. It should be `Roboto Medium`.